app/main/views/g_cloud.py

```diff
@@ -622,6 +630,31 @@ def end_search(framework_family, project_id):
     ), 200 if not errors else 400
 
 
+@direct_award.route('/<string:framework_family>/projects/<int:project_id>', methods=['POST'])
+def update_project(framework_family, project_id):
+    """Generic view for bits of the task list that need to POST stuff."""
```

Comment:
This doesn't feel like a particularly generic view if all it can do is set `readyToAssess`?

Reply:
Well, the URI is generic and it's referenced from a `<form>` that that entirely wraps the page. Potentially the same view could be used for other interactive or ajaxy things going on on that page.

I was just hoping to encourage people not to make another view that did the same thing. Can reword if necessary.